src/http-spec.ts

```diff
@@ -61,15 +61,15 @@ export interface IHttpOperation<Bundle extends boolean = false> extends INode, I
     ? IHttpOperationResponse<true> | (Pick<IHttpOperationResponse, 'code'> & Reference)
     : IHttpOperationResponse<false>)[];
   servers?: IServer[];
-  callbacks?: IHttpCallbackOperation[];
+  callbacks?: (Bundle extends true ? IHttpCallbackOperation[] | Reference : IHttpCallbackOperation<false>)[];
   security?: HttpSecurityScheme[][];
   deprecated?: boolean;
   internal?: boolean;
 }
 
-export type IHttpCallbackOperation = Omit<IHttpOperation<false>, 'servers' | 'security' | 'callbacks'> & {
+export interface IHttpCallbackOperation<Bundle extends boolean = false> extends IHttpOperation {
```

Comment:
```suggestion
export interface IHttpCallbackOperation<Bundle extends boolean = false> extends Omit<IHttpOperation<Bundle>, 'security' | 'callbacks'> {
```

Reply:
or perhaps keep security if we actually store it (which we may do since http-spec executes transformOas{2,3}Operation under the hood)
```suggestion
export interface IHttpCallbackOperation<Bundle extends boolean = false> extends Omit<IHttpOperation<Bundle>, 'callbacks'> {
```

Reply:
I updated it to `export interface IHttpCallbackOperation<Bundle extends boolean = false> extends Omit<IHttpOperation<Bundle>, 'callbacks'> {`